wallet-jni/Cargo.toml

```diff
@@ -0,0 +1,13 @@
+[package]
+name = "wallet-jni"
+version = "0.1.0"
+authors = ["Daniel Sanchez Quiros <daniel.sanchez@iohk.io>"]
+edition = "2018"
+
```

Comment:
missing license

Reply:
Good catch. Same licence as in the others?

Reply:
yes